packages/runtime/src/useCases/common/RuntimeErrors.ts

```diff
@@ -96,18 +96,18 @@ class Relationships {
     public wrongRelationshipStatus(relationshipId: string, status: string): ApplicationError {
         return new ApplicationError(
             "error.runtime.relationships.wrongRelationshipStatus",
-            `The relationship '${relationshipId}' has the wrong status (${status}) to run this operation`
+            `The relationship '${relationshipId}' has the wrong status ('${status}') to run this operation`
```

Comment:
-"Relationship" instead of "relationship".
- Please end this sentence with a full stop.